odoo/addons/base/rng/common.rng

```diff
@@ -24,6 +24,8 @@
                         <rng:value>replace</rng:value>
                     </rng:choice>
                 </rng:attribute>
+                <!-- Allows to take a custom View widget for handling. Defined in view_manager. -->
+                <rng:optional><rng:attribute name="js_class"/></rng:optional>
```

Comment:
Not sure that it should be written there.
For which view type do you need it?

Reply:
For the list view.
But https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/js/chrome/view_manager.js#L343
implies that attribute should be kinda generic

Reply:
Generic for all views, yes, but only for the root node. I'm not expert of rng definitions, but it looks like you tolerate it on every nodes.

Reply:
@aab-odoo 
Indeed, ref=overload is pretty much used everywhere
and there is no define name=generic_view

So, I modified the PR